src/qtiItem/core/Loader.js

```diff
@@ -48,113 +76,147 @@ var Loader = Class.extend({
                 ret = _.union(ret, this.getRequiredClasses(data[i]));
             }
         }
+
         return ret;
     },
-    loadRequiredClasses: function(data, callback, reload) {
-        var i;
-        var requiredClass,
-            requiredClasses = this.getRequiredClasses(data, reload),
-            required = [];
+    loadRequiredClasses(data, callback, reload) {
+        let requiredClass;
+        const requiredClasses = this.getRequiredClasses(data, reload);
+        const required = [];
 
-        for (i in requiredClasses) {
+        for (let i in requiredClasses) {
             requiredClass = requiredClasses[i];
             if (this.classesLocation[requiredClass]) {
                 required.push({
-                    module : this.classesLocation[requiredClass],
+                    module: this.classesLocation[requiredClass],
                     category: 'qti'
                 });
             } else {
-                throw new Error('missing qti class location declaration : ' + requiredClass);
+                throw new Error(`missing qti class location declaration : ${requiredClass}`);
             }
         }
 
         moduleLoader([], () => true).addList(required).load().then(loadeded => {
-            loadeded.forEach( QtiClass => {
+            loadeded.forEach(QtiClass => {
                 this.qti[QtiClass.prototype.qtiClass] = QtiClass;
             });
             callback.call(this, this.qti);
         });
     },
-    getLoadedClasses: function() {
+    getLoadedClasses() {
         return _.keys(this.qti);
     },
-    loadItemData: function(data, callback) {
-        var _this = this;
-        _this.loadRequiredClasses(data, function(Qti) {
-            var i;
+    loadItemData(data, callback) {
+        this.loadRequiredClasses(data, (Qti) => {
```

Comment:
single parameter does not need parenthesis 

```suggestion
        this.loadRequiredClasses(data, Qti => {
```